From 62c2f041890a6e20770350721a0a2138d0b38634 Mon Sep 17 00:00:00 2001
From: Vincent Davis Jr <vince@underview.tech>
Date: Sat, 3 Dec 2022 23:35:51 -0600
Subject: [PATCH] libavcodec: fix v4l2_req_devscan.h

Upstream-Status: Inappropriate

RPI-Distro repo clones original ffmpeg and applies patches to enable
raspiberry pi support.

Fixes minor differences between v4l2_req_devscan.c
and v4l2_req_devscan.h after all patches have been
applied.

Signed-off-by: Vincent Davis Jr <vince@underview.tech>
---
 libavcodec/v4l2_req_devscan.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libavcodec/v4l2_req_devscan.h b/libavcodec/v4l2_req_devscan.h
index 0baef365..cd9c49ac 100644
--- a/libavcodec/v4l2_req_devscan.h
+++ b/libavcodec/v4l2_req_devscan.h
@@ -1,6 +1,8 @@
 #ifndef _DEVSCAN_H_
 #define _DEVSCAN_H_
 
+#include <stdint.h>
+
 struct devscan;
 struct decdev;
 enum v4l2_buf_type;
@@ -13,7 +15,8 @@ const char *decdev_video_path(const struct decdev *const dev);
 enum v4l2_buf_type decdev_src_type(const struct decdev *const dev);
 uint32_t decdev_src_pixelformat(const struct decdev *const dev);
 
-const struct decdev *devscan_find(struct devscan *const scan, const uint32_t src_fmt_v4l2);
+const struct decdev *devscan_find(struct devscan *const scan,
+                                  const uint32_t src_fmt_v4l2);
 
 int devscan_build(void * const dc, struct devscan **pscan);
 void devscan_delete(struct devscan **const pScan);
-- 
2.38.1

